shared/teams/team/invites/invite-history.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as Constants from '../../../constants/teams'
+import * as Types from '../../../constants/types/teams'
+import * as TeamsGen from '../../../actions/teams-gen'
+import {memoize} from '../../../util/memoize'
+import {Section} from '../../../common-adapters/section-list'
+import {formatDurationLong} from '../../../util/timestamp'
+import {useTeamDetailsSubscribe} from '../../subscriber'
+import {ModalTitle} from '../../common'
+
+type Props = Container.RouteProps<{teamID: Types.TeamID}>
+
+const splitInviteLinks = memoize((inviteLinks: Set<Types.InviteLink>): {
+  active: Array<Types.InviteLink>
+  expired: Array<Types.InviteLink>
+} => ({active: [...inviteLinks].filter(i => !i.expired), expired: [...inviteLinks].filter(i => i.expired)}))
+
+const InviteHistory = (props: Props) => {
+  const teamID = Container.getRouteProps(props, 'teamID', Types.noTeamID)
+  useTeamDetailsSubscribe(teamID)
+  const teamDetails = Container.useSelector(s => Constants.getTeamDetails(s, teamID))
+  const loading = teamDetails === Constants.emptyTeamDetails // TODO should be a better way to check this
```

Comment:
```
const {loadingTeam, teamDetails} = Common.useTeamDetailsWithSubscribe(teamID)

useTeamDetailsWithSubscribe = (teamID) => {
useTeamDetailsSubscribe(teamID);   
const teamDetails = Container.useSelector(s => Constants.getTeamDetails(s, teamID))
  const loadingTeam = teamDetails === Constants.emptyTeamDetails
return {teamDetails, loadingTeam}
}
```
WDYT?

Reply:
the object equality part seems fragile. i'll just pull directly from the map for now